ci/testsuite-result.json

```diff
@@ -210,10 +207,9 @@
     "command": "zone set_ns example.org ns2.example.org",
     "command_filter": null,
     "command_filter_negate": false,
-    "command_issued": "zone set_ns example.org ns2.example.org # requires force because ns2 is unknown",
+    "command_issued": "zone set_ns example.org ns2.example.org  #  requires force because ns2 is unknown",
```

Comment:
A little bit unsure why this changed. I guess we ignore whitespace somewhere, and we never had a diff until now that caused this command to be considered "different".